share/errmsg-utf8.txt

```diff
@@ -18724,6 +18724,9 @@ ER_BINLOG_CANT_READ_INDEX
 ER_RPL_FAILED_IN_RLI_INIT_INFO
   eng "Failed in %s called from Relay_log_info::rli_init_info()."
 
+ER_DB_ADMISSION_CONTROL
```

Comment:
This should replace ER_PLACEHOLDER_50028. In the original 5.6 code, it came after ER_DB_READ_ONLY.